Gulpfile.js

```diff
@@ -265,6 +278,124 @@ gulp.task('test-client-travis', ['build'], function () {
 
 gulp.task('travis', [process.env.GULP_TASK || '']);
 
+//Documentation
+gulp.task('lint-docs', function () {
+    function lintFiles (files) {
+        return new Promise(function (resolve, reject) {
+            var config = {
+                'files':  files,
+                'config': require('./.markdownlint.json')
+            };
+
+            markdownlint(config, function (err, result) {
+                var lintErr = err || result && result.toString();
+
+                if (lintErr)
+                    reject(lintErr);
+                else
+                    resolve();
+            });
+        });
+    }
+
+    return globby('./docs/articles/**/*.md').then(lintFiles);
+});
+
+gulp.task('clear-site', ['lint-docs'], function (cb) {
```

Comment:
I was thinking in a different way: we have nothing to do until docs are not okay, So, lint is executed at first, without a parallel task.